### ISSUE TITLE
healpix links against old version of cifitsio 

### ISSUE THREAD

**Issue by kjmeagher** [Homebrew/homebrew-science#4597]
healpix bottle links against `libcfitsio.2.dylib` which is not provided by the latest version of cfitsio. 
```
 otool -L  /usr/local/opt/healpix/lib/libhealpix_cxx.0.dylib /usr/local/opt/healpix/lib/libhealpix_cxx.0.dylib:
	/usr/local/opt/healpix/lib/libhealpix_cxx.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/cfitsio/lib/libcfitsio.2.dylib (compatibility version 2.0.0, current version 2.3.37)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1225.1.1)
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 120.1.0)
```
compiles fine with `--build-from-source` so it probably just needs a new bottle.

**Comment by ilovezfs** [comment]
Feel free to open a revision bump PR.

**Comment by iMichka** [comment]
I opened a PR for that here (with a version bump): #5124

**Comment by iMichka** [comment]
Should be fixed with a06f907.
